### PR TITLE
fix typos in Jetson Orin documentation

### PR DESCRIPTION
**Pull request by penguinland**
The docs didn't quite match up with the Jetson Orin on my desk:
- Pins 3 and 5 are I2C bus 7, as shown by plugging something into those pins and running `i2cdetect -y -a -r 7`
- Pins 27 and 28 are I2C bus 1, as shown by an analogous check.
- I did _not_ swap the I2C `jetson-io.py` IDs, since they look correct (based on running `jetson-io.py` myself).
- The SPI bus is number 0, as found by reading from an ADC plugged into it. I'm happy to share my config if you want; setting this up is more complicated than I'd prefer.
- Things in the `/dev` column should start with slashes, to show that they're absolute paths.
- I changed the device names to the nearest things that actually exist in `/dev`. Are those the right devices? I'm unsure; our code doesn't interact with them directly. but they're the devices that exist on my Orin which are the closest to what was already in the docs, and I trust whoever wrote the previous version. 

I've never made a PR in this repo before. Once it's merged, what else do I need to do to get it deployed?

### CODE REVIEW

**Review by sguequierre — COMMENTED** [review]
Thank you for making a pr and for testing this out, these changes look good to me! I added the previous version of this table to this document in conjunction with @thegreatco from his setup notes, so I am going to ask him for review on this as well. I think the /dev path changes are accurate but I am not 100% sure about the bus IDs switch-up.

Oh and re. deployment --> once this is merged it should be deployed to production automatically!

**Comment by penguinland** [comment]
This branch became out of date because someone else merged something else. I've rebased it, and will continue rebasing as necessary. There's still just the single commit.

**Review by sguequierre — APPROVED** [review]
LGTM but would suggest waiting for Pete to review/approve when he gets back from holiday before merging.

**Comment by thegreatco** [comment]
I had opened https://viam.atlassian.net/browse/DOCS-262 during my investigation. I found that Pins 3/5 were i2c bus 1 and 27/28 were i2c bus 7. I guess my notes were wrong or something changed with nvidia?

**Review by thegreatco — APPROVED** [review]

**Comment by penguinland** [comment]
My guess is that the notes were wrong because the notes are how I'd design the system myself and they seem perfectly reasonable. :stuck_out_tongue:

**Pull request merged**
